### ISSUE TITLE
time.sleep returning instantly

### ISSUE THREAD

**Issue by eah13** [skulpt/skulpt#584]
time.sleep is resolving instantly in the lastest versions of skulpt

Try this code on skulpt.org:

```python
import time
start = time.time()
time.sleep(1)
finish = time.time()
print(finish-start)
```
I get ~`.001` instead of ~`1.001`

Perhaps related to #579?  It's as if `Sk.setTimeout` isn't getting set correctly in env.js

**Comment by rixner** [comment]
You merged #579, did you not test it first?  It worked for me...  Does it work in your deployment?

**Comment by eah13** [comment]
I don't believe actual timing is tested in the test suite, so I missed it.  I didn't notice this regression til we prepared to deploy.

**Comment by rixner** [comment]
So, I assume that means it doesn't work in your deployment either?

After some investigation, it looks like ```setTimeout``` wants ```this``` to be ```Window```.  By reassigning it this way, ```this``` is ```Sk``` when it is invoked.

I don't quite understand why it worked for me, but it did.  Maybe there are browser-specific behaviors?

Anyway, does it work for you if you change line 64 of env.js to be:

```Sk.setTimeout = function(func, delay) { setTimeout(func, delay); };```

**Comment by eah13** [comment]
> So, I assume that means it doesn't work in your deployment either?

in the process of upgrading to 0.11 we found this bug in a dev environment and paused.  We've not deployed to prod yet so trinket.io is unaffected.

> browser-specific

I've seen it in Firefox and Chrome

> Anyway, does it work for you if you change line 64....

No it does not.  It's taken me a second to get my bearings on `time`, partially because I use a cloud env and didn't immediately realize there was no `setTimeout` at all without a browser.  So even with pre-regression code I get the current in-browser behavior (no sleeping) when I `run` but not when I `brun`.  I checked out a few random commits just to be sure but the regression happens as of 3695bac (the commit of yours I merged. sorry - overconfidence in tests!).

I still can't figure out why Sk.setTimeout isn't happening as expected.  Haven't spent a ton of time on this just yet- hoping someone with more familiarity with the time module might see a simple reason.

**Comment by albertjan** [comment]
It might be interesting for us if we can move our runtime from v8 to node, as we do have a setTimeout in node and it would make maintaining the version a lot easer too.

**Comment by eah13** [comment]
Agreed.  Updated this issue to apply only to the demo on Skulpt.org since that's the only place it still exists as far as I know.

**Comment by prescod** [comment]
The problem is on Skulpt.org but to be more precise it is in:

http://www.skulpt.org/static/skulpt.min.js

If I refer to that file, my program handles time.sleep improperly. If I refer instead to a local skulpt.min.js (and skulpt-stdlib.js) then it works fine.

This is a bit of a problem because skultp.org advocates new users point to those files by URL in its sample programs. What would be the process for getting those files updated?

**Comment by schnorea** [comment]
Work around for broken sleep if pulling from CDN.  Use turtle.
```
import turtle
t = turtle.Turtle()

print("Hello")
t.forward(100)  # This is approx a 1 second delay
print("World")
```
